src/java.base/share/classes/java/nio/MappedByteBuffer.java

```diff
@@ -221,6 +222,9 @@ public final MappedByteBuffer load() {
      * mapping modes. This method may or may not have an effect for
      * implementation-specific mapping modes. </p>
      *
+     * @throws UncheckedIOException
+     *         If the underlying operation fails
+     *
      * @return  This buffer
      */
     public final MappedByteBuffer force() {
```

Comment:
The 2-arg force method will need to be updated too.
Given that the methods are specified to write the buffer content back to the underlying storage device then I think it should be okay to say "If an I/O occurs writing the ...".

Reply:
Will change.